opensearch_py_ml/ml_models/test_question_answering.py

```diff
@@ -0,0 +1,37 @@
+# Save our model as pt
```

Comment:
Why do we need this file here?

Reply:
We should assert if our answer string is equal to the official answer. I can add an assertion statement instead of printing the two outputs!

Reply:
Yeah that would be awesome. I would like you to present tomorrow about this model tracing in our office hour so that we can learn about this together.